### ISSUE TITLE
[Question] Best way to show css of a story component that was generated by css-in-js(styled/emotion)

### ISSUE THREAD

**Issue by cybervaldez** [storybookjs/storybook#12965]
I want to use story builder as an actual component designer but unfortunately show code only displays the component with passed args. Anyone knows how I could show css as well?

**Comment by shilman** [comment]
You can override the source completely using the `docs.source.code` story parameter https://storybook.js.org/docs/react/writing-docs/doc-blocks#source

**Comment by steve-vuori** [comment]
I am also looking for the best way to output the CSS Source on a component for designers to review.

**Comment by iftimiedaniel** [comment]
> You can override the source completely using the `docs.source.code` story parameter https://storybook.js.org/docs/react/writing-docs/doc-blocks#source

@cybervaldez 
That's correct. In order to do that you need to import your CSS code and pass it to the `source.code` of your `<Canvas>`.
Create a ` .mdx` file doc for your component:

```
/* Button.mdx */
import { Meta, Canvas } from '@storybook/blocks';
import * as ButtonStories from './Button.stories';
import ButtonCss from './button.css';

<Meta of={ButtonStories} />

<Canvas 
    of={ButtonStories.Primary} 
    sourceState="shown" 
    withToolbar="true" 
    source={{
        code: ButtonCss ,
        language: 'css'
    }} 
/>
```
![Screenshot_1](https://github.com/storybookjs/storybook/assets/12616638/4a694ccf-9294-41b4-94e7-62f4e0fbdba6)

**Comment by shaman-apprentice** [comment]
I ended up with the following (make sure to have `raw-loader` installed as dev dependency): 

```mdx
// button.stories.mdx
import { Meta, Source} from '@storybook/blocks';
import ButtonCss from '!raw-loader!./button.css';

<Meta title="Button" /> 

<Source type="code" language="css" code={ButtonCss } />
```
